src/ambianic/config_mgm/__init__.py

```diff
@@ -0,0 +1,3 @@
+
+from .config_diff import ConfigChangedEvent, Config, ConfigList, ConfigDict, ConfigChangedEvent
```

Comment:
This is really minor, but it seems to make sense. ConfigChangeEvent is imported twice. I'll fix this.